src/transformers/modeling_flax_auto.py

```diff
@@ -0,0 +1,169 @@
+# coding=utf-8
+# Copyright 2018 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Auto Model class. """
+
+
+import logging
+from collections import OrderedDict
+
+from .configuration_auto import AutoConfig, BertConfig, RobertaConfig
+from .configuration_utils import PretrainedConfig
+from .modeling_flax_bert import FlaxBertModel
+from .modeling_flax_roberta import FlaxRobertaModel
+
+
+logger = logging.getLogger(__name__)
```

Comment:
Centralized logging uitl
```suggestion
logger = logging.get_logger(__name__)
```